onmt/train/Checkpoint.lua

```diff
@@ -76,23 +76,81 @@ function Checkpoint:saveEpoch(validPpl, epochState, verbose)
   self:save(filePath, info)
 end
 
+-- structural parameters - the parameters with true can be changed dynamically in a train_from
+local structural_parameters = {
```

Comment:
yes I agree -but the current implementation was even less future proof :smile: with hard coded set-up of options. I think we do need 3 values - I will take the point.